frontend/src/components/HomePage/Filters.css

```
@@ -1,6 +1,7 @@
 .filters-bar {
   display: inline-flex;
   flex-direction: row;
+  gap: 30px;
   position: sticky;
   top: 0;
   width: 100%;
```

```suggestion
  width: 100%;
  background-color: #FBFAEF;
```
so that the bills do not show behind the inputs on scroll